File: src/test/resources/output/python/requests/full.py
import requests

url = "http://mockbin.com/har"

querystring = {
  "foo": [
    "bar",
    "baz"
  ],
  "baz": "abc",
  "key": "value"
}

payload = "foo=bar"
headers = {"Cookie":"foo=bar; bar=baz","content-type":"application/x-www-form-urlencoded","accept":"application/json"}

response = requests.request("POST", url, data=payload, headers=headers, params=querystring)

print(response.text)

File: src/test/resources/output/javascript/jquery/full.js
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://mockbin.com/har?baz=abc&foo=bar&foo=baz&key=value",
  "method": "POST",
  "headers": {
    "Cookie": "foo=bar; bar=baz",
    "content-type": "application/x-www-form-urlencoded",
    "accept": "application/json"
  },
  "data": {
    "foo": "bar"
  }
}

$.ajax(settings).done(function (response) {
  console.log(response);
});
